src/frontend/web/src/ratingpage/RatingCard.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card, Avatar } from 'antd';
+
+import 'antd/dist/antd.css';
+import './RatingCard.scss';
+import EffortBar from '../components/EffortBar';
+
+const RatingCard = ({ about, age, effort, faculty, imgSrc, name, semester,
+    times, workPartition }) => (
+        <Card className="rating-card">
+            <div dir="rtl">
```

Comment:
We should have this on the whole website - I think it is better to remove it from here and add it globally (I think I did it)